halyard/halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/SpringConfig.java

```diff
@@ -82,8 +82,12 @@ private OAuth2Security populateOAuth2Security(OAuth2 oauth2) {
     registration.put("scope", oauth2.getClient().getScope());
     registration.put(
         "clientAuthenticationScheme", oauth2.getClient().getClientAuthenticationScheme());
-    registration.put(
-        "redirect-uri", "\"" + oauth2.getClient().getPreEstablishedRedirectUri() + "\"");
+
+    if (oauth2.getClient().getPreEstablishedRedirectUri() != null
```

Comment:
This change seems unrelated.  Is there a test covering this scenario?

Reply:
No, I will add the test. When `redirect-uri ` is null the end value will be `"null"` so I am checking for null before adding the value.

Reply:
added tests here https://github.com/spinnaker/spinnaker/pull/7052/commits/caaf5dd2a1e60f61101c53f4646729f7aaf6d2cf